style.css

```diff
@@ -0,0 +1,851 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap');
+
+/* GENERAL STYLING */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.flex-container {
+  display: flex;
+  align-items: center;
+}
+
+.button {
+  border: 3px solid #ec5242;
+  padding: 5px 10px;
+  color: #ec5242 !important;
+}
+
+/* Navigation Bar */
+.upper-nav {
+  display: none;
+}
+
+.second-list li {
+  padding: 19px;
+  display: flex;
+}
+
+.sec-navbar span {
+  display: block;
+}
+
+.close-btn {
+  display: block;
+  margin-top: 0;
+  position: fixed;
+}
+
+.sec-navbar {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Please adjust the spacing between the `navbar `elements to match the design guidelines as according to the requirements we must follow the layout of the provided [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 

|Cindy shin's design|Your design|
|---|----|
|<img width="1501" alt="Screenshot 2023-05-29 at 12 29 19 PM" src="https://github.com/kit0-0/Capstone-summit/assets/44798044/7c16d36e-6b8b-45ea-840c-cb9f65350d79">|<img width="1501" alt="Screenshot 2023-05-29 at 12 24 47 PM" src="https://github.com/kit0-0/Capstone-summit/assets/44798044/3684dbc2-f8c1-4310-9af8-192fc2f995a9">|